docs/dev/guidelines/client-tests.rst

```diff
@@ -63,6 +63,9 @@ Some guidelines:
 1. A component should be tested in isolation without any dependencies if possible. Do not simply import the whole production module. Only import real dependencies if it is essential for the test
    that the real dependency is used. Instead, use mock pipes, mock directives and mock components that the component under test depends upon. A very useful technique is writing `stubs for child components <https://angular.io/guide/testing-components-scenarios#stubbing-unneeded-components>`_.
    This has the benefit of being able to test the interaction with the child components.
+2. According to [Chai](https://www.chaijs.com/api/bdd) you should make expectations as specific as possible. This means
```

Comment:
If you introduce 2. you should increment the existing points accordingly, i.e. 

L209: **3.** Do not use ``NO_ERRORS_SCHEMA`` (`link <https://angular.io/guide/testing-components-scenarios#no_errors_schema>`_). (now still 2.)
L212: **4.** Sinon uses sandboxes, which remove the need of keeping track of every fake created, which greatly simplifies cleanup and improves readability. (now still 3.)

and so on... 
I assume you just forgot about that :)

Reply:
Indeed. They were so far down I completely missed that something comes after that 😅